### ISSUE TITLE
Legend shortcut does not work with German keyboard layout

### ISSUE THREAD

**Issue by m99coder** [UseMacro/macro-chrome-extension#9]
The advertised shortcut `alt`+`/` does not open the legend. Neither on https://usemacro.github.io/# nor on any of the supported web pages like Google and Github. I’m using a German keyboard layout which might be the reason for this issue.

**Comment by carlosflorencio** [comment]
I'm using the Portuguese keyboard and can't make it work either.
Can we add the shortcut: option + / (numpad) ?

**Comment by andyzg** [comment]
You're right, the reason for this is because the key for `/` actually sends `-` as an event. We'll need to transform characters depending on the keyboard, which might be a bit tricky to do. Will look into it 😃
